source/System.cpp

```diff
@@ -253,7 +260,8 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 					}
 			}
 			else if(child.Size() >= 4)
-				asteroids.emplace_back(type, child.Value(valueIndex + 1), child.Value(valueIndex + 2));
+				asteroids.emplace_back(type, child.Value(valueIndex + 1), child.Value(valueIndex + 2),
+					child.Value(valueIndex + 3) ? child.Value(valueIndex + 3) : 1);
```

Comment:
I'm not quite sure if this will work, test it.
```suggestion
					(child.Size() > 4) ? child.Value(valueIndex + 3) : 1);
```